packages/module/src/SourceDetailsMenuItem/SourceDetailsMenuItem.scss

```diff
@@ -0,0 +1,28 @@
+.pf-chatbot__source-details-subhead {
+  color: var(--pf-t--global--text--color--subtle);
+  font-size: var(--pf-t--global--icon--size--font--xs);
+  font-weight: 500;
+}
+
+.pf-chatbot__source-details-heading {
+  font-size: var(--pf-t--global--font--size--body--default);
+  word-wrap: break-word;
+}
+
+.pf-chatbot__source-details-icon {
+  width: 100%;
+}
+
+// this is only used in demo code
+.pf-chatbot__source-details-icon > .pf-v6-c-icon__content > svg {
```

Comment:
Hey! It looks like the icon color is the same on default and on hover - Can you change it to be pf-t--global--icon--color--status--custom--default on default and pf-t--global--icon--color--status--custom--hover on hover